dags/dag_functions.py

```diff
@@ -129,3 +129,10 @@ def task_fail_slack_alert(
         proxy=proxy,
     )
     return failed_alert.execute(context=context)
+
+def get_readme_docmd(readme_path, dag_name):
+    import re
```

Comment:
This is debatable. This makes DAG loading faster, but not too organized or in a standard way. I'm not sure actually what's the best way to do it 🤔

Reply:
I added a detailed comment to make this more clear. I think it will make our DAGs more readable.

Reply:
sorry I wasn't clear about this. I meant having import statements inside functions

Reply:
Ohhhh ok, I'll move it to the top.